src/tests/generators.test.js

```diff
@@ -0,0 +1,9 @@
+const { expect } = require('chai')
+const { arrayToStringList, pascalCase } = require('../generators/utils')
+
+describe('When I use utils', () => {
+  it('Test arrayToStringList', () => {
+    const result = arrayToStringList(['ASDSAd', 'ASDSAd', 'ASDSAd', 'ASDSAd'])
```

Comment:
can you add more readable and friendly strings here?

Also please add more readable name instead "When i use utils"

Reply:
Done @jhomarolo